.github/dependabot.yaml

```diff
@@ -1,9 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: "docker"
-    directories:
-      -  "/images/**"
-    schedule:
-      interval: "daily"
-    labels:
-    - "skip-review"
```

Comment:
no human review?

Reply:
Thankfully that was not working anyhow. See for example: https://github.com/kubernetes/test-infra/pull/36061